frontend/src/lib/components/Errors/stackFrameLogic.tsx

```diff
@@ -0,0 +1,30 @@
+import { kea, path } from 'kea'
+import { loaders } from 'kea-loaders'
+import api from 'lib/api'
+
+import type { stackFrameLogicType } from './stackFrameLogicType'
+
+export interface StackFrame {
+    filename: string
+    lineno: number
+    colno: number
+    function: string
+    in_app?: boolean
+}
+
+export const stackFrameLogic = kea<stackFrameLogicType>([
```

Comment:
Checking myself - this isn't used anywhere, yeah? For now it doesn't need to be I guess, since we're not using the context yet

Reply:
Correct, dropped the line context stuff in favour of getting this out first. Will double back on this later